docs/cedarling/cedarling-policy-store.md

```diff
@@ -21,49 +21,88 @@ The JSON schema looks like this:
 
 ```
 {
-    "some_unique_identifier": {
-        "name": "...",
-        "description": "...",
-        "policies": {},
-        "identity_source": {},
-        "schema": "..."
-    }
+    "policystore_id": "...",
+    "policies": {...},
+    "schema": "...",
+    "trusted_issuers": [],
+    "cedar_version": "v2.4.7"
 }
+
 ```
 
+- **policystore_id** : (String, no spaces) The unique identifier for the policy store.
+- **policies** : (Json) Json object containing one or more policy_id as key and policy object as value
```

Comment:
Maybe you should specify that policy content is in base64 as well.
You can add the json structure, like the "description", "creation_date" ...